dev-libs/libcgroup/files/cgconfig.confd

```diff
@@ -1,4 +1,7 @@
 # /etc/conf.d/cgconfig: config file for /etc/init.d/cgconfig
```

Comment:
Please don't rename those files, it's going to be utterly confusing and make reading `git log` a PITA. Just remove old files if they're no longer used.